index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <title>Flappy Bird Game</title>
+    <script src="main.js"></script>
+    <style>
+        #Flappy{
+            border: 1px solid #949494;
```

Comment:
Any particular reason you made border in color `#949494` instead of `gray` or `black`?

Reply:
For no reason, I change to black.